ui/app/mainui/AppMain.qml

```diff
@@ -73,12 +73,12 @@ Item {
         }
         onOpenDownloadModalRequested: {
             const downloadPage = downloadPageComponent.createObject(appMain,
-                {
-                    newVersionAvailable: newVersionJSON.available,
-                    downloadURL: newVersionJSON.url,
-                    currentVersion: rootStore.profileSectionStore.getCurrentVersion(),
-                    newVersion: newVersionJSON.version
-                })
+                                                                    {
+                                                                        newVersionAvailable: newVersionJSON.available,
+                                                                        downloadURL: newVersionJSON.url,
+                                                                        currentVersion: rootStore.profileSectionStore.getCurrentVersion(),
+                                                                        newVersion: newVersionJSON.version
+                                                                    })
```

Comment:
This looks off. Is this intended?

Reply:
like CMD+S I automatically use CMD+A/CMD+I which call qml formatting :) I will fix